sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -331,17 +331,11 @@ def generate_account_sas(
     :param permission:
         The permissions associated with the shared access signature. The
         user is restricted to operations allowed by the permissions.
-        Required unless an id is given referencing a stored access policy
```

Comment:
Just making this note here but applies to all `generate_account_sas`. Stored access policy is **not** available for account SAS, and so going to clean up the docstring from mentioning using stored access policy.